stdlib/2and3/nis.pyi

```diff
@@ -0,0 +1,11 @@
+# Stubs for nis (Python 3.6)
+# Unix only
```

Comment:
This comment could be replaced by wrapping the rest of the code in
```py
if sys.platform != 'win32':
```

Reply:
Thanks for your good advice!